src/state/user/hooks/index.tsx

```diff
@@ -266,3 +268,27 @@ export function useTrackedTokenPairs(): [Token, Token][] {
     return Object.keys(keyed).map((key) => keyed[key])
   }, [combinedList])
 }
+
+export const useWatchlistTokens = (): [string[], (address: string) => void] => {
```

Comment:
Update to existing code:
New hooks in `user` state to add/remove watchlist token/pool